src/destroy.ts

```diff
@@ -88,4 +88,19 @@ async function run() {
 
 }
 
-run()
+// Run the main function only if the AWS Creds are set.
+(async () => {
```

Comment:
I _think_ you don't need to `async` this self invoking function since all the actions within it happen synchronously.